integration_tests/test_commit.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+from git_wrapper import exceptions
+from git_wrapper.commit import GitWrapperCommit
+
+
+def test_apply_diff(repo_root):
+    commit = GitWrapperCommit(repo_root)
+    test_branch = "test_apply_diff"
+
+    # Create a diff file
```

Comment:
I would consider pulling this off to a data file and then read it in at test time. You could us something like pytest_datadir (https://github.com/gabrielcnr/pytest-datadir) to easily find the file. We use pytest-datadir in atkinson and koji-wrapper for testing.